From f7a01c6db6f3aaf161b1c057633a4e53adf8b62b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kristian=20H=C3=B8gsberg?= <krh@bitplanet.net>
Date: Fri, 11 Oct 2013 16:40:39 -0700
Subject: [PATCH 35/39] Revert "Export CompositeRedirectSubwindows and
 CompositeUnRedirectSubwindows"

This reverts commit 15a7e62a9848f2089180c49b02887565d29bb896.

Now that we don't redirect windows on behalf of the wm, we don't need this.
---
 composite/compalloc.c    | 11 -----------
 composite/compositeext.h |  4 ----
 2 files changed, 15 deletions(-)

diff --git a/composite/compalloc.c b/composite/compalloc.c
index 98b27b1..b7d731e 100644
--- a/composite/compalloc.c
+++ b/composite/compalloc.c
@@ -46,7 +46,6 @@
 #endif
 
 #include "compint.h"
-#include "compositeext.h"
 
 static void
 compScreenUpdate(ScreenPtr pScreen)
@@ -412,11 +411,6 @@ compRedirectSubwindows(ClientPtr pClient, WindowPtr pWin, int update)
     return Success;
 }
 
-int CompositeRedirectSubwindows (WindowPtr pWin, int update)
-{
-    return compRedirectSubwindows (serverClient, pWin, update);
-}
-
 /*
  * Free one of the per-client per-subwindows resources,
  * which frees one redirect per subwindow
@@ -488,11 +482,6 @@ compUnredirectSubwindows(ClientPtr pClient, WindowPtr pWin, int update)
     return BadValue;
 }
 
-int CompositeUnRedirectSubwindows (WindowPtr pWin, int update)
-{
-    return compUnredirectSubwindows (serverClient, pWin, update);
-}
-
 /*
  * Add redirection information for one subwindow (during reparent)
  */
diff --git a/composite/compositeext.h b/composite/compositeext.h
index a072966..0b148f0 100644
--- a/composite/compositeext.h
+++ b/composite/compositeext.h
@@ -34,10 +34,6 @@
 extern _X_EXPORT Bool CompositeRegisterAlternateVisuals(ScreenPtr pScreen,
                                                         VisualID * vids,
                                                         int nVisuals);
-extern _X_EXPORT int CompositeRedirectSubwindows(WindowPtr pWin,
-						 int update);
-extern _X_EXPORT int CompositeUnRedirectSubwindows (WindowPtr pWin,
-						    int update);
 
 extern _X_EXPORT RESTYPE CompositeClientWindowType;
 
-- 
1.8.3.1

